app/dashboard/models.py

```diff
@@ -827,6 +827,41 @@ def additional_funding_summary_sentence(self):
             sentence += f" worth ${afs['usd_value']}"
         return sentence
 
+    def creation_to_dict(self):
+        return {
+            # info to xfr over from self as override fields (this is because sometimes
```

Comment:
is there not a django object to dict() method that we could just use?

Reply:
There's a `django.forms` utility method to do this...  I suppose I can use that.

Reply:
@owocki Updated to use `django.form.models.model_to_dict`